lib/pages/class_feedback/view/classes_feedback_checklist.dart

```diff
@@ -0,0 +1,177 @@
+import 'package:acs_upb_mobile/authentication/service/auth_provider.dart';
+import 'package:acs_upb_mobile/pages/class_feedback/service/feedback_provider.dart';
+import 'package:acs_upb_mobile/pages/class_feedback/view/class_feedback_view.dart';
+import 'package:acs_upb_mobile/pages/classes/model/class.dart';
+import 'package:acs_upb_mobile/pages/classes/service/class_provider.dart';
+import 'package:acs_upb_mobile/widgets/error_page.dart';
+import 'package:acs_upb_mobile/widgets/scaffold.dart';
+import 'package:acs_upb_mobile/widgets/toast.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class ClassFeedbackChecklist extends StatefulWidget {
+  const ClassFeedbackChecklist({Key key, this.classes}) : super(key: key);
+  final Set<ClassHeader> classes;
+
+  @override
+  _ClassFeedbackChecklistState createState() => _ClassFeedbackChecklistState();
+}
+
+class _ClassFeedbackChecklistState extends State<ClassFeedbackChecklist> {
+  Map<String, dynamic> classesFeedback = {};
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  Future<void> fetchFeedbackProvided() async {
```

Comment:
Nit: Completed sounds better I think (also the adjective should come before the noun)